aws/vpc/main.tf

```diff
@@ -0,0 +1,29 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "3.11.0"
+
+  name                 = "vpc-${var.cluster_name}"
+  cidr                 = var.cidr
+  azs                  = local.aws_availability_zones
+  private_subnets      = var.private_subnets
+  public_subnets       = var.public_subnets
+  enable_nat_gateway   = true
+  single_nat_gateway   = true
+  enable_dns_hostnames = true
+
+  public_subnet_tags = {
+    "kubernetes.io/cluster/${var.cluster_name}" = "shared"
+    "kubernetes.io/role/elb"                    = "1"
+  }
+
+  private_subnet_tags = {
+    "kubernetes.io/cluster/${var.cluster_name}" = "shared"
+    "kubernetes.io/role/internal-elb"           = "1"
+    "tier"                                      = "private"
+  }
+}
+
+locals {
+  // zones supported by our kind of machine type in our region
+  aws_availability_zones = ["us-east-1a", "us-east-1b", "us-east-1c", "us-east-1d", /*"us-east-1e",*/ "us-east-1f"]
```

Comment:
why `us-east-1e` is in comment? remove or add it

Reply:
I took all files from our repo - I think this zone doesn't support this server type. I will remove it